### PR TITLE
Conv2d padding test data

### PR DESCRIPTION
**Pull request by Abdurrahheem**
This PR add model, data and generating script for this PR.

### CODE REVIEW

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -1581,6 +1581,17 @@ def forward(self, x):
...
+class Conv2D(nn.Module):
+    def __init__(self, in_chanl, out_chanl, kernel_size, stride=1, padding=0):
+        super(Conv2D, self).__init__()
+        self.layer = nn.Conv2d(in_chanl, out_chanl, kernel_size, stride, padding)
+    def forward(self, x):
+        return self.layer(x)
+
+model = Conv2D(3, 6, 3, 1, 1)
+x = torch.randn(1, 3, 30, 30)
+save_data_and_model("conv2dPadding", x, model, export_params=True)
```
Comment (opencv-alalek) [inline testdata/dnn/onnx/generate_onnx_models.py:1593]: all lowercase names are preferable for files.
